tests/test_utils.py

```diff
@@ -431,6 +431,29 @@ def test_get_dict_from_mapping(self):
                 self.assertEqual(value, result_with_mapping_object[key])
                 self.assertEqual(value, result_with_dict[key])
 
+    def test_converter_get_prefixes_including_synonyms_and_get_subconverter(self):
+        """Test if get_subconverter() works with prefix synonyms."""
+        EPM = [
+            {
+                "prefix": "SCTID",
+                "prefix_synonyms": ["snomed"],
+                "uri_prefix": "http://snomed.info/id/",
+            },
+        ]
+
+        converter = Converter.from_extended_prefix_map(EPM)
+        self.assertNotIn("snomed", converter.get_prefixes(include_synonyms=False))
+        self.assertIn("snomed", converter.get_prefixes(include_synonyms=True))
```

Comment:
it's not SSSOM-py's job to test `curies` functionality. The test should show how the change in this PR actually effects input and output to SSSOM-py functions

Reply:
This is not a test, it's some code to show why the pr the way it is should not be merged.. I don't intend to add this as a test here